homeassistant/components/fyta/sensor.py

```diff
@@ -33,9 +34,7 @@
 class FytaSensorEntityDescription(SensorEntityDescription):
     """Describes Fyta sensor entity."""
 
-    value_fn: Callable[[str | int | float | datetime], str | int | float | datetime] = (
-        lambda value: value
-    )
+    value_fn: Callable[[Plant], StateType | datetime] = lambda value: value.name
```

Comment:
```suggestion
    value_fn: Callable[[Plant], StateType | datetime]
```
Everything has one now, you do need to change the dataclass to `@dataclass(frozen=True, kw_only=True)`

Reply:
This is what I had in my first draft as well. But then pytest complained (TypeError: non-default argument 'value_fn' follows default argument).

Reply:
Because of the missing kw_only=True?

Reply:
Yes, this did the trick. Thnx.